docs/glossary.md

```diff
@@ -14,27 +14,27 @@ To help you navigate the platform more effectively, familiarize yourself with th
+ |                                                                                                                           |**Environment** - a logical entity that facilitates the inspection of application sets. Each environment is linked to a distinct Kubernetes namespace dedicated to containerized workloads, ensuring a one-to-one mapping between the environment and the namespace. This structure allows for the sequential promotion of applications from one environment to the next.|
  |                                                                                                                           |**Codebase** - business entity that possesses a code.|
  |                                                                                                                           |**Codebase Branch** - business entity that represents a specific version in a Git branch. Every codebase branch has a Codebase Docker Stream entity.|
- |                                                                                                                           |**Codebase Docker Stream** - a deployable component that leads to the application build and displays that the last build was verified on the specific stage. Every CD pipeline stage accepts a set of Codebase Image Streams (cbis) that are input and output. _SAMPLE: if an application1 has a master branch, the input cbis will be named as [app name]-[pipeline name]-[stage name]-[master] and the output after the passing of the DEV stage will be as follows: [app name]-[pipeline name]-[stage name]-[dev]-[verified]._|
+ |                                                                                                                           |**Codebase Docker Stream** - serves as a key component in managing the deployment lifecycle of an application version. It facilitates the identification and progression of application versions that are ready for deployment. Within the Deployment Flow, each environment leverages Codebase Image Streams (CBIS) to manage application versions effectively. For instance, an application named 'application1' associated with a master branch would be initially tagged in the CBIS using the format: `[codebase name]-[branch name]`, e.g., 'application1-main'. As the application version progresses through the Deployment Flow, reaching readiness for deployment in a specific environment, the CBIS naming convention evolves to indicate its deployment flow and target environment, typically following the structure: `[deployment flow name]-[environment name]-[codebase name]`. This approach ensures a clear and systematic tracking of application versions from their initial identification through to their readiness for deployment in various environments within the deployment pipeline.|
```

Comment:
Codebase Image Stream

Reply:
flow and target environment, following the structure: `[deployment flow name]-[environment name]-[codebase name]-verified`